include/GoogleSync/GoogleSync.php

```diff
@@ -740,19 +875,56 @@ public function delEvent($event, $meeting_id)
         $statusCode = $return->getStatusCode();
 
         if ($statusCode >= 200 && $statusCode <= 299) {
-         // 2xx statusCode = success
-            $GLOBALS['log']->debug(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Received Success Status Code: ' . $statusCode . ' on delete.');
+            // 2xx statusCode = success
+            $this->logger->debug(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Received Success Status Code: ' . $statusCode . ' on delete.');
 
             // This removes the gsync_id reference from the table.
-            $sql = "UPDATE Meetings SET gsync_id = '' WHERE id = '" . $meeting_id . "'";
+            $sql = "UPDATE meetings SET gsync_id = '' WHERE id = '" . $meeting_id . "'";
             $res = $this->db->query($sql);
+            if (!$res) {
+                $this->logger->fatal(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Failed to remove gsync_id from record' . $meeting_id);
+            }
 
             $this->syncedList[] = $meeting_id;
             return true;
         } else {
-            $GLOBALS['log']->fatal(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Received Failure Status Code: ' . $statusCode . ' on delete!');
+            $this->logger->fatal(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Received Failure Status Code: ' . $statusCode . ' on delete!');
+            return false;
+        }
+    }
+
+    /**
+     * Helper function. Clear all popup reminders from crm meeting
+     *
+     * @param string $event_id The ID of the event in the DB
+     *
+     * @return bool Success/Failure
+     */
+    public function clearPopups($event_id)
+    {
+        if (!isset($event_id) || empty($event_id)) {
+            $this->logger->fatal(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Event_id is missing');
             return false;
         }
+
+        // Disable all popup reminders for the SuiteCRM meeting
+        $eventIdQuoted = $this->db->quoted($event_id);
+        $sql = sprintf("UPDATE reminders SET popup = '0' WHERE related_event_module_id = '%s' AND deleted = '0'", $eventIdQuoted);
+        $res = $this->db->query($sql);
+        if (!$res) {
+            $this->logger->fatal(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'SQL Failure!');
+            return false;
+        }
+
+        // Mark all reminders where both popup and email are disabled as deleted.
+        $sql = sprintf("UPDATE reminders SET deleted = '1' WHERE popup = '0' AND email = '0' AND related_event_module_id = '%s' AND deleted = '0'", $eventIdQuoted);
```

Comment:
Can you please resolve this query and previous one above (https://github.com/salesagility/SuiteCRM/pull/6539/files#diff-034e880ea433e2cc2571088f7cbc6620R912) in one step.

Reply:
I don't know that it's possible. The first query sets all popups to 0 for the reminders. The next one sets reminders with both popups and emails set to zero as deleted. It's likely that the reminder has an email.

Reply:
The next commit will have a refactored pushPullSkip method.